#goal definition

uint8 segment_id
SegmentState state
float32 duration
---
#result definition
bool success
---
#feedback
